client/my-sites/checkout/composite-checkout/use-create-payment-methods.js

```diff
@@ -291,7 +301,7 @@ function useCreateExistingCards( { storedCards, stripeConfiguration } ) {
 	return existingCardMethods;
 }
 
-export default function useCreatePaymentMethods( {
+export default function useCreatePaymentMethodsForCheckout( {
```

Comment:
There's nothing wrong with this, and the name is more clear, but I'd prefer that if we do this we also rename the file to match. It's just my pedantic brain wanting default exports to match their file name to make them easier to find later.

Alternatively, we could just leave this name in its old version. The other functions in this file could also be better named (`useCreateCreditCard` is really `useCreateCreditCardPaymentMethod`) and maybe that's a whole PR in itself?

Reply:
I decided to just revert the change, since renaming the exports here probably requires more careful thought than we really need to spend at the moment. :)